### PR TITLE
v6.19.10

### PR DESCRIPTION
**Pull request by DanailH**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,52 @@
...
+
+### Date Pickers
+
+#### `@mui/x-date-pickers@6.19.10`
+
+No changes.
+
+#### `@mui/x-date-pickers-pro@6.19.10` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')
+
+Same changes as in `@mui/x-date-pickers@6.19.10`.
```
Comment (flaviendelangle) [inline CHANGELOG.md:38]: That's probably more correct

```suggestion
No changes.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,52 @@
...
+
+### Date Pickers
+
+#### `@mui/x-date-pickers@6.19.10`
+
+No changes.
+
+#### `@mui/x-date-pickers-pro@6.19.10` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')
+
+No changes.
```
Comment (LukasTy) [inline CHANGELOG.md:38]: Just checking in: has this been agreed upon?
AFAIK, we haven't released packages in v6 if they didn't have any changes. 🤔

Comment (DanailH) [reply CHANGELOG.md:38]: I think we agreed to release everything that Lerna suggests, which is the current case :D

Comment (LukasTy) [reply CHANGELOG.md:38]: I'm a bit curious why Lerna suggested releasing packages which did not receive any changes... 🤷

Comment (flaviendelangle) [reply CHANGELOG.md:38]: No idea why :laughing: 

But we must clearly move on this discussion to have an agreement on a good release strategy

**Review by flaviendelangle — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (LukasTy) [reply CHANGELOG.md:38]: I've switched to v6.x branch, ran `yarn install` and then `yarn release:version`.
This is the result:
Screenshot 2024-04-12 at 16 45 08

Comment (DanailH) [reply CHANGELOG.md:38]: I did the same, even nuked by yarn cache. I can revert the other packages

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,52 @@
...
+
+### Charts / `@mui/x-charts@6.19.10`
+
+No changes.
+
+### Tree View / `@mui/x-tree-view@6.19.10`
+
+No changes.
+
```
Comment (LukasTy) [inline CHANGELOG.md:47]:
```suggestion
```

**Review by LukasTy — APPROVED** [review]
🚀 💯

**Pull request merged**